docs/releases/2.5.1.rst

```diff
@@ -0,0 +1,74 @@
+Translate Toolkit 2.5.1
+***********************
+
+*Released on 25 April 2020*
+
+This release contains improvements and bug fixes.
+
+
+Changes
+=======
+
+Formats and Converters
+----------------------
+
+- PO
+
+    - Avoid adding extra space on empty comment lines
+    - Several performance improvements
+
+- Android resources
+
+    - Workaround broken plural handling for languages missing other tag
+    - Fix setting rich content with comments
+
+- YAML
+
+    - Fixed handling of empty keys
+    - Improved round trip preserving of comments and style
```

Comment:
Missing 2ad462e ?

Reply:
This is more a bugfix for  IMHO the issue was introduced by cbf4c9c766474f9433bc988ad0818d9c522ec5ce.